### PR TITLE
Document how to connect to the testnet

### PR DESCRIPTION
**Pull request by raduom**
Add a short documentation about how to configure the node to connect to any testnet, and also include a troubleshooting section since people have been having issues connecting to the testnets and it may be worth to have some sort of a knowledge base somewhere.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

**Review by sjoerdvisscher — APPROVED** [review]

File: plutus-chain-index/notes/ConnectingToTheTestnet.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# How to connect to a testnet
+
+## Get `cardano-node` onto the path
+
+The easiest way of doing this is to enter a `nix-shell` that puts the node on the path.
```
Comment (j-mueller) [inline plutus-chain-index/notes/ConnectingToTheTestnet.md:5]: Do we not have to run `cabal build cardano-node` in the nix shell?

File: plutus-chain-index/notes/ConnectingToTheTestnet.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### 1. Synchronisation stops after a number of blocks.
+
+I was getting this error message:
+
+```
+[𝝺:cardano.node.DnsSubscription:Error:111] [2021-09-23 09:25:36.12 UTC] Domain: "relays.alonzo-purple.dev.cardano.org" Application Exception: 3.64.236.32:3001 HeaderError (At (Block {blockPointSlot = SlotNo 2261160, blockPointHash = f9dee539ef2f498021a0ff5bb522849da957a33e81be5f01d9bd687140d81b37})) (HeaderEnvelopeError (OtherHeaderEnvelopeError (HardForkEnvelopeErrFromEra S (S (S (S (Z (WrapEnvelopeErr {unwrapEnvelopeErr = ObsoleteNodeCHAIN 6 5})))))))) (Tip (SlotNo 2259021) db47db2f7d0f667f4203fbd242da8cb089994cccebe77134563304330eb692cf (BlockNo 104553)) (Tip (SlotNo 2907080) 5fe3cdde5510898c0d324557b7040f1f486ce84058778a181c28249daab44405 (BlockNo 131683))
+```
+
+This was due to using an old version of the node, which does not support the required protocol version.
```
Comment (j-mueller) [inline plutus-chain-index/notes/ConnectingToTheTestnet.md:37]: Where can we find the current version of the node? Just use the latest tagget version from the node repo?

Comment (raduom) [reply plutus-chain-index/notes/ConnectingToTheTestnet.md:37]: It's not trivial. What I did is I nix-shell-ed into `cardano-node` master, built it and started it. Should I include this information?

File: plutus-chain-index/notes/ConnectingToTheTestnet.md (hunk 1 above)
Comment (raduom) [reply plutus-chain-index/notes/ConnectingToTheTestnet.md:5]: You need to cabal build all and then exec. I will add it.

File: plutus-chain-index/notes/ConnectingToTheTestnet.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# How to connect to a testnet
+
+## Get `cardano-node` onto the path
```
Comment (silky) [inline plutus-chain-index/notes/ConnectingToTheTestnet.md:3]:
```suggestion
## Get `[cardano-node](https://github.com/input-output-hk/cardano-node)` onto the path
```

File: plutus-chain-index/notes/ConnectingToTheTestnet.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# How to connect to a testnet
+
+## Get `[cardano-node](https://github.com/input-output-hk/cardano-node)` onto the path
+
+The easiest way of doing this is to enter a `nix-shell` then build the node and use `cabal exec` to run it.
```
Comment (j-mueller) [inline plutus-chain-index/notes/ConnectingToTheTestnet.md:5]: am I running this in the plutus repository or in the cardano-node one?

Comment (raduom) [reply plutus-chain-index/notes/ConnectingToTheTestnet.md]: Updated the document with the answer.

File: plutus-chain-index/notes/ConnectingToTheTestnet.md (hunk 2 above)
Comment (raduom) [reply plutus-chain-index/notes/ConnectingToTheTestnet.md]: I updated the document with this information also.

**Review by j-mueller — APPROVED** [review]

**Pull request merged**